src/Drawer/Drawer.react.js

```diff
@@ -34,10 +34,12 @@ class Drawer extends PureComponent {
         const { children } = this.props;
 
         const styles = getStyles(this.props, this.context);
+        const keyboardShouldPersistTaps = this.props.keyboardShouldPersistTaps || "never";
+        const keyboardDismissMode = this.props.keyboardDismissMode || "none";
```

Comment:
Could we use ScrollView's default values if they are not present in this.props? In an other words - could you remove these two lines? :) And also use line 34?

const { keyboardShouldP..., children } = this.props;